resources/templates/messages.html

```diff
@@ -0,0 +1,43 @@
+<html>
+<body>
+
+<form action="/create-message" method="post">
+    <input type="text" maxlength="140" placeholder="Enter your text" name="messageText"/>
+    <button type ="send">Send</button>
+</form>
+
+
+
+
+
+
+
+
+
+</form>
+    <ol>
+    {{#messages}}
+    <li>{{.}}</li>
+    {{/messages}}
+    </ol>
+
+<form action="/edit" method="post">
+    <button type="edit">edit</button>
+</form>
+
```

Comment:
Everything looks good up until edit and delete.  You need to add some text fields to the forms on messages.html.  then add matching routes in Main.java.  If you need a hand with the logic on how to get it to delete or edit just ask.  Again everything looks good so far.